### PR TITLE
:lipstick: adding toast notifications in make report page

### PR DESCRIPTION
**Pull request by syamilu**
Im adding toaster from react-hot-toast, for satisfaction and cosmetic purposes tho haha. Also adding check for email in form

### CODE REVIEW

**Comment by syamilu** [comment]
Here screenshot for the make report page responsiveness fix

image

image

File: src/client/pages/MakeReportPage.tsx [hunk 1]
```diff
@@ -11,27 +12,33 @@ export default function MakeReportPage() {
     console.log(formData);
     // console.log(data.image)
     try {
+      const toastLoad = toast.loading("Submitting report ...");
       const res = await fetch("http://localhost:3000/api/submit-report", {
         method: "post",
         body: formData,
       });
-
+      await new Promise((resolve) => setTimeout(resolve, 3000)).then(() => {
```
Comment (ElyasAsmad) [inline src/client/pages/MakeReportPage.tsx:22]: 🗿🗿🗿

Comment (hakimnazry24) [reply src/client/pages/MakeReportPage.tsx:22]: 🤣

Comment (syamilu) [reply src/client/pages/MakeReportPage.tsx:22]: HAHAHAHA will fix it later GAGAGA

**Pull request merged**